CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ This file tries to follow recommendations of [Keep a CHANGELOG](http://keepachan
 
 ## [Unreleased]
 
+## [0.1.9] - 2019-03-22
+
+### Added
+- Delete application plan customization
+- Delete activedocs
+- List accounts
+- Delete account
+- Delete application
+- Show OIDC conf
```

Comment:
no need to abbreviate `configuration` to `conf`, right? This is a document for humans.